sources/svg/gridicons-list-ordered-rtl.svg

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="iso-8859-1"?>
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><rect x="0" fill="none" width="24" height="24"/>
+    <g id="Page-1" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
```

Comment:
Do some cleanup to the source before final build & merge. :) 
Details on how to do it are here: https://github.com/Automattic/gridicons/wiki/Guidelines-for-SVG-Sources